numba_cuda/numba/cuda/cudadrv/nvrtc.py

```diff
@@ -372,19 +372,25 @@ def compile(src, name, cc, ltoir=False):
         f"-I{get_cuda_paths()['include_dir'].info}",
     ]
 
+    nvrtc_version = nvrtc.get_version()
+    nvrtc_ver_major = nvrtc_version[0]
+
     cudadrv_path = os.path.dirname(os.path.abspath(__file__))
     numba_cuda_path = os.path.dirname(cudadrv_path)
-    numba_include = f"-I{numba_cuda_path}"
+
+    numba_include = (
+        f"-I{os.path.join(numba_cuda_path, 'include', str(nvrtc_ver_major))}"
```

Comment:
This will reference a non-existent folder as soon as there's a CUDA 13 release - can we just select 11 if the NVRTC version is 11 and 12 otherwise?